addons/web_editor/static/src/xml/editor.xml

```diff
@@ -47,7 +47,7 @@
     <!-- Media Dialog (allows to choose an img/pictogram/video) -->
     <div t-name="web_editor.dialog.media">
         <ul class="nav nav-tabs" role="tablist">
-            <li class="nav-item"><a role="tab" href="#editor-media-image" aria-controls="editor-media-image" data-toggle="tab" t-attf-class="nav-link#{widget.noImages ? ' d-none' : ''}">Image</a></li>
+            <li class="nav-item"><a role="tab" href="#editor-media-image" aria-controls="editor-media-image" data-toggle="tab" t-attf-class="nav-link active#{widget.noImages ? ' d-none' : ''}">Image</a></li>
```

Comment:
Supposed to be automatically active thanks to https://github.com/odoo/odoo/blob/master/addons/web_editor/static/src/js/widgets/widgets.js#L1157 -> the fix would be to check why it is not the case

Reply:
@qsm-odoo 
there is only one case when we click in image button in the top panel ( https://drive.google.com/file/d/1PZEtCLqVTQkVcJihHxD4GaTF2SIOjYfH/view?usp=drivesdk ) then image tab-panel is not active.